ratpack-spring-boot/src/main/java/ratpack/spring/config/RatpackConfiguration.java

```diff
@@ -51,8 +51,8 @@ public void run(String... args) throws Exception {
     server.start();
   }
 
-  @PreDestroy
-  public void stop() throws Exception {
```

Comment:
Do we need to leave the stop() method since it was public and have it just call destroy if called but we can remove the predestroy?

Reply:
Ah, that's a good point. Yeah we should probably do that.